container-dependency-versions/pom.xml

```diff
@@ -475,7 +475,8 @@
         <!-- and then verify  by doing: ' ls -l vespa/vespa_jersey2/target/dependency' -->
         <hk2.version>2.5.0-b32</hk2.version>
         <hk2.osgi-resource-locator.version>1.0.1</hk2.osgi-resource-locator.version>
-        <jackson2.version>2.8.4</jackson2.version>
```

Comment:
The naming scheme is inconsistent. Consider changing both version parameters to either have `jackson` or `jackson2` as prefix.

Reply:
The 'jackson2' is a remains from when we had both jackson versions at the same time. Let's try to rename it in a later PR.